### PR TITLE
[HOTFIX](ci/cd) fix: CD for dbt compile + fix requirements

### PR DESCRIPTION
**Pull request by lmontier-pass**
# Hotfix

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I have updated the dag
- [ ] I will create a review on slack. The review task should be short (<10min).

### CODE REVIEW

**Review by cdelabre — APPROVED** [review]

File: .github/workflows/deploy_composer.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
     uses: ./.github/workflows/reusable_deploy_composer.yml
     needs: [test-orchestration, test-jobs, linter]
     with:
-      TARGET_ENV:  "staging"
+      TARGET_ENV:  "stg"
```
Comment (valoumiaou) [inline .github/workflows/deploy_composer.yml:78]: staging

**Review by valoumiaou — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_composer.yml (hunk 1 above)
Comment (lmontier-pass) [reply .github/workflows/deploy_composer.yml]: ça casse si on ne fait pas ça...

File: orchestration/airflow/orchestration-requirements.in [hunk 2]
```diff
@@ -5,10 +5,10 @@ sshtunnel~=0.4
...
 pytest-cov~=4.0
 mock~=4.0.2
-#apache-airflow-providers-ssh~=3.7
-#apache-airflow-providers-mysql~=5.1
-#apache-airflow-providers-postgres~=5.2
-#google-cloud-secret-manager~=2.16
+apache-airflow-providers-ssh~=3.7
+apache-airflow-providers-mysql~=5.1
+apache-airflow-providers-postgres~=5.2
+google-cloud-secret-manager~=2.16
```
Comment (lmontier-pass) [inline orchestration/airflow/orchestration-requirements.in:11]: en fait elles sont imposées dans les contraintes. Pour moi ça ne sert à rien

Comment (cdelabre) [reply orchestration/airflow/orchestration-requirements.in:11]: Les tests fails sinon :/

**Pull request merged**